postgres/initdb/02-init.sql

```diff
@@ -26,7 +26,7 @@ create table versions (
   id uuid primary key,
```

Comment:
in het algemeen: moet er van al deze wijzigingen nog iets benoemd worden in de changelog?

Reply:
Eh ja, dat concept (de 'changelog') is er net pas bijgekomen. :-)
Dus .. laten we dat zeker doen.

Reply:
CHANGELOG.md updated in https://github.com/knaw-huc/textrepo/commit/12281484c2f87c999df3694e94bfc5b497e7c6d7